scripts/migrations/common.py

```diff
@@ -101,3 +106,41 @@ def get_aws_client(resource, *, role_arn):
         aws_secret_access_key=credentials["SecretAccessKey"],
         aws_session_token=credentials["SessionToken"],
     )
+
+
+def get_storage_client(api_url):
```

Comment:
This can be cached as well – the storage service class has its own logic for refreshing tokens. You don't need a new instance unless the on-disk credentials have changed.